app/routines/create_user.rb

```diff
@@ -54,7 +54,12 @@ def guessed_first_name(full_name)
 
   def guessed_last_name(full_name)
     return nil if full_name.blank?
-    full_name.split("\s").drop(1).join(' ')
+    parts = full_name.split("\s")
+    if parts.many?
+      parts.drop(1).join(' ')
+    else
+      full_name
+    end
```

Comment:
If full_name is just one "Jimmy", this change means `guessed_last_name` will be `"Jimmy"` whereas before it was `""`.  What was the reason for the change?